wlc/__init__.py

```diff
@@ -132,36 +133,42 @@ def process_error(error):
                 "HTTP error {0}: {1} {2}".format(status_code, reason, error_string)
             )
 
-    def raw_request(self, method, path, params=None, files=None):
+    def raw_request(self, method, path, data=None, files=None, params=None):
```

Comment:
We should perhaps add data as the last parameter to reduce chances of incorrect invocation in existing calls or customization.

Reply:
So this is a little hard to see in the git log, but essentially what's happening here are 2 changes:

1. Rename existing parameter "params" to "data" (because this "data" parameter is what ultimately gets passed to requests as either the "data" or "json" value.
1. Introduce new parameter, "params" (because this value is what is ultimately passed to requests as the "params" value)

I realize that there might be some backwards compatibility issues for downstream libraries, but I get the impression that wlc is primarily meant as a CLI, not as a python api (just considering their documentation).